template/requirements/docs.in

```diff
@@ -10,3 +10,6 @@ sphinx-autodoc-typehints
 sphinx-copybutton
 sphinx-design
 platformdirs<4  # temporary until virtualenv has release with support for this
+# Temporary until questionary (dep of copier) updates
```

Comment:
pip-compile-multi requires that all envs resolve to the same versions of all packages.

Without this pin, In dev.txt, we get the pinned version here because of copier. In docs.txt we get a newer version because ipython depends on it but does not exclude the newer version. So pip-compile-multi fails to resolve.

This is the same issue as with platformdirs in the line above.

Reply:
Hmm, I feel having a dev and affect the pinning is actually a problem: We may end up in situations where a dev-only dependency causes the use of a certain version, hiding problems in CI.

Can we avoid this? Should `dev.in` be moved elsewhere? Can it be handled using a different call to `pip-compile-multi`?

Reply:
My initial intention was to make dev reflect the other envs to make it easier to work with. So it should have the same versions.

I'm not sure this really is an issue. We anyway don't see the exact same versions in the test/docs/etc envs as in production.